simulation/policies.py

```diff
@@ -24,49 +24,57 @@ def p_randomly(params, step, sL, s, **kwargs):
             "new_participant_investment": None,
             "new_participant_tokens": None
         }
+        dict_ans = {}
 
         arrival_rate = (1+sentiment)/config.arrival_rate_denominator
-        if probability_func(arrival_rate):
-            ans["new_participant"] = True
```

Comment:
This is no longer needed. The policy should return an array of dict objects (new participants) with properties `new_participant_investment` and `new_participant_tokens`